Portal/ViewModels/BaledWithWireModel.cs

```diff
@@ -0,0 +1,18 @@
+using EPRN.Common.Enums;
+using EPRN.Portal.Resources;
+using System.ComponentModel.DataAnnotations;
+
+namespace Portal.ViewModels
+{
+    public class BaledWithWireModel
+    {
+        public int JourneyId { get; set; }
+
+        public string? WasteType { get; set; }
+
+        public bool? BaledWithWire { get; set; }
+
+        [Required(ErrorMessageResourceName = "ErrorMessage", ErrorMessageResourceType = typeof(BaledWithWireResource))]
+        public string BaledWithWireScreen { get; set; }
```

Comment:
Is this property required if you have already got one of type bool on line 13? I would have thought that it would be something like:

`[Required(ErrorMessageResourceName = "ErrorMessage", ErrorMessageResourceType = typeof(BaledWithWireResource))]`
`public bool? BaledWithWire { get; set; }`

Reply:
Done.